api/src/main/webapp/WEB-INF/cron_default.yaml

```diff
@@ -23,7 +23,7 @@ cron:
     Update our database cache of users' two-factor authentication settings on their GSuite accounts
     via Google Directory Service.
   url: /v1/cron/bulkSyncTwoFactorAuthStatus
-  schedule: every 24 hours
+  schedule: every 1 hours
```

Comment:
Whose decision and what was the rationale? I don't see a reason to run this hourly. Here was my feedback on the spreadsheet for this cron:

"""
Again, not super time sensitive. 24h seems fine. I could be argued up to every 6h if it made things easier to maintain.
"""

Reply:
I must have pasted into the wrong cell.